Support adding dependencies to an existing Ktor project (#44)
ktorio/ktor-cli: README.md

ktor openapi -o path/to/project petstore.yaml
```

## Add a Ktor dependency to an existing project

To add a Ktor dependency to a Gradle project in the current working directory, use the `add` command:
```shell
ktor add server-core
```

> Use `-p` or `--project` option to specify a path to the project directory:

Reviewer: ```suggestion
Use the `-p` or `--project` option to specify a path to the project directory:
```

```shell
ktor add -p /path/to/project server-core
```

> Multiple modules can be added with a single command:

Reviewer: ```suggestion
You can add multiple modules with a single command:
```

```shell
ktor add -p /path/to/project server-core client-core json
```

> Currently, a Ktor dependency can be added only to **non**-multiplatform Gradle projects with Kotlin DSL.

Reviewer: ```suggestion
Currently, Ktor dependencies can only be added to **non-multiplatform** Gradle projects using Kotlin DSL.
```
